index.html

```diff
@@ -258,34 +258,52 @@ <h1 class="font-weight-bold pb-3">SOURCES</h1>
         var start = moment("2018-01-01");
         var percentageBudgetLeft = 0;
 
+        /**
+         * Format a number to string format, with commas to separate groups of thousands.
+         */
         function numberFormater(numberToFormat) {
```

Comment:
I just noticed that there's a typo, should be called `numberFormatter()` with two T. Would you mind fixing this?